php-apache/app/public/index.php

```diff
@@ -1,6 +1,6 @@
 <?php
 try {
-    $dbh = new PDO('mysql:dbname=[ 6 ];host=[ 7 ];unix_socket=/tmp/mysql.sock', 'root', '[ 8 ]');
+    $dbh = new PDO('mysql:dbname=docker_review;host=docker_review_review-mysql_1;unix_socket=/tmp/mysql.sock', 'root', 'e2QNqCUd');
```

Comment:
[imo]

こちらコンテナの起動時に自動で割り当てられるコンテナ名になります。
起動状況によっては例えば、`docker_review_review-mysql_2` というようになってしまい、そうなってしまうと、Congratulations と表示されなくなってしまうかと思います。

docker-compose.yml に定義しているコンテナ名をそのまま host に指定すると良いかと思います。

```suggestion
    $dbh = new PDO('mysql:dbname=docker_review;host=review-mysql;unix_socket=/tmp/mysql.sock', 'root', 'e2QNqCUd');
```

Reply:
正直よくわからないまま書いていました。
自動で割り当てられているコンテナ名だったんですね。。